docs/README.md

```diff
@@ -406,6 +399,13 @@ These configs are needed for the web app.
       <td></td>
       <td><a href="https://docs.stripe.com/webhooks#endpoint-secrets" target="_blank">Stripe webhook key</a> for enabling payments</td>
     </tr>
+    <tr>
+      <td><code>USER_VERIFICATION_ENABLED</code></td>
+      <td>false</td>
+      <td>boolean</td>
+      <td><code>true</code></td>
```

Comment:
Should default to false, not true (we only want this feature on the main managed service)